### PR TITLE
Added parameter distribution container

### PR DESCRIPTION
**Pull request by HansKallekleiv**
Added ParameterDistribution container that shows multiple ensembles of model parameters in histogram/boxplot

### CODE REVIEW

File: webviz_subsurface/containers/__init__.py [hunk 1]
```diff
@@ -33,11 +34,12 @@
...
+__all__ = ['ParameterDistribution',
            'DiskUsage',
            'SubsurfaceMap',
            'HistoryMatch',
            'Intersect',
            'MorrisPlot',
            'InplaceVolumes',
-           'ReservoirSimulationTimeSeries']
+           'ReservoirSimulationTimeSeries',
+           'ParameterCorrelation']
```
Comment (anders-kiaer) [inline webviz_subsurface/containers/__init__.py]: Unnecessary diff in the end?

**Comment by HansKallekleiv** [comment]
> * Should/could `_parameter_correlation` also use the `_fmu_input.load_parameters` introduced here?

Yes. Perhaps better to handle in a separate PR? Could also attempt to make some of the other containers use this code. Currently I believe the same ensemble / ensemblesets are loaded multiple times in different containers.

**Comment by HansKallekleiv** [comment]
> * Add option to normalize to percent instead of count?

Skipping this one unless seen as important. The plot resets when changing mode. Would require adding uirevision to the figure layout...
